schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/process/add-extent-from-geokeywords.xsl

```diff
@@ -36,7 +36,7 @@
   <xsl:variable name="replaceMode"
                 select="geonet:parseBoolean($replace)"/>
   <xsl:variable name="serviceUrl"
-                select="concat($gurl, '/keywords?pNewSearch=true&amp;pTypeSearch=2&amp;pKeyword=')"/>
+                select="concat(substring($gurl, 1, string-length($gurl)-4), '/api/0.1/registries/vocabularies/search.xml?q=')"/>
```

Comment:
You can remove from the API path the version number (this was removed in 4.0.x).

Reply:
Fixed in latest commit!